Assets/Editor/Resource/RecourcesSpawnerEditor.cs

```diff
@@ -16,8 +20,41 @@ public override void OnInspectorGUI()
 
             ResourceSpawner spawner = (ResourceSpawner)target;
 
-            if (GUILayout.Button("Spawn Resources"))
-                spawner.InitializedSpawning();
+            if (spawner.SpawnAreas.Count <= 0)
+                return;
+
+            EditorGUILayout.BeginHorizontal();
+
+            if (GUILayout.Button("All (Re)Spawn Resources"))
+                spawner.SpawnAllResources();
+
+            if (GUILayout.Button("All Remove Resources"))
+                spawner.DestoryAllResources();
+
+            EditorGUILayout.EndHorizontal();
+
+            _selectNumber = EditorGUILayout.Popup("Select an option:", _selectNumber, SetToStringList(spawner.SpawnAreas));
+
+            EditorGUILayout.BeginHorizontal();
+
+            if (GUILayout.Button($"(Re)Spawn resources in {spawner.SpawnAreas[_selectNumber].Area.gameObject.name}"))
+                spawner.SpawnResourcesInArea(_selectNumber);
+
+            if (GUILayout.Button($"Remove resources in {spawner.SpawnAreas[_selectNumber].Area.gameObject.name}"))
+                spawner.DestroyAreaResources(_selectNumber);
+
+            EditorGUILayout.EndHorizontal();
```

Comment:
Slight nitpick but looks more clear 

```suggestion
            EditorGUILayout.BeginHorizontal();
            {
                  if (GUILayout.Button("All (Re)Spawn Resources"))
                      spawner.SpawnAllResources();
      
                  if (GUILayout.Button("All Remove Resources"))
                      spawner.DestoryAllResources();
            }
            EditorGUILayout.EndHorizontal();

            _selectNumber = EditorGUILayout.Popup("Select an option:", _selectNumber, SetToStringList(spawner.SpawnAreas));

            EditorGUILayout.BeginHorizontal();
            {
                    if (GUILayout.Button($"(Re)Spawn resources in {spawner.SpawnAreas[_selectNumber].Area.gameObject.name}"))
                        spawner.SpawnResourcesInArea(_selectNumber);
        
                    if (GUILayout.Button($"Remove resources in {spawner.SpawnAreas[_selectNumber].Area.gameObject.name}"))
                        spawner.DestroyAreaResources(_selectNumber);
            }
            EditorGUILayout.EndHorizontal();
```